histoDB/histodb-dataserver/src/main/java/eu/itesla_project/histodb/server/CimHistoImporter.java

```diff
@@ -40,27 +42,21 @@
     private ITeslaDatasource datasource;
 
     private static final ComputationManager computationManager;
-
-    private static final Importer importerCim;
-    private static final Importer importerXml;
     private static final List<Importer> importers;
 
     static {
         try {
-            importers=new ArrayList<>();
             computationManager = new LocalComputationManager();
-            importerCim = Importers.getImporter("CIM1", computationManager);
-            if (importerCim!=null) {
-                importers.add(importerCim);
-            } else {
-                log.warn("CIM importer implementation not found");
-            }
-            importerXml = Importers.getImporter("XIIDM", computationManager);
-            if (importerXml!=null) {
-                importers.add(importerXml);
-            } else {
-                log.warn("iidm-xml importer implementation not found");
-            }
+            importers = Stream.of("CIM1", "XIIDM", "UCTE")
+                    .map(importerID -> {
+                        Importer importer = Importers.getImporter(importerID, computationManager);
+                        if (importer == null) {
+                            log.warn(importerID + " importer implementation not found");
+                        }
+                        return importer;
+                    })
+                    .filter(Objects::nonNull)
+                    .collect(Collectors.toList());
```

Comment:
Consider Importers.list() method to be fully independant of the available importer plugins in the classpath, to automatically handle new importer plugins (CGMES, IEEE..) without any modification and to simplify the static initializer